geomesa-convert/geomesa-convert-common/src/main/scala/org/locationtech/geomesa/convert/Transformers.scala

```diff
@@ -226,7 +226,7 @@ object Transformers extends EnhancedTokenParsers with LazyLogging {
       if (idx == -1) {
         idx = ctx.indexOf(n)
       }
-      ctx.get(idx)
+      if (idx < 0) null else ctx.get(idx)
```

Comment:
or if "idx == -1" was "doEval == null", which is a bit clearer, basically saying "if we don't have a doEval defined then define it")